### ISSUE TITLE
ERR: invalid SDP

### ISSUE THREAD

**Issue by QuantumEntangledAndy** [aler9/rtsp-simple-server#41]
When streaming I receive this message

```
ERR: invalid SDP: sdp: invalid syntax `m=`
```

Followed by a disconnect. This is probably client specific as it works from ffmpeg just not from my ios app.

It may be related to #24 #8 and in that issue you requested a wireshark dump. So I attach it here: 

[stream_dump.pcap.zip](https://github.com/aler9/rtsp-simple-server/files/4936696/stream_dump.pcap.zip)

p.s. sorry its a zip, gitub seems to not like the `pcap` extention.

**Comment by aler9** [comment]
Hello, thank you very much for providing the dump, that is all i need to fix this bug.
Recently the SDP library was changed to https://github.com/pion/sdp, that is much more supported and shared with my favourite Go WebRTC implementation, but i discovered it supports only SDP with explicit timestamp, ie with a 
```
t=0 0
```

before the
```
m=video 0 RTP/AVP/TCP 96
```

So i'll add it in case it is not present.

**Comment by QuantumEntangledAndy** [comment]
I'm glad I could help. Let me know when I can test it and I'll have a go :)

**Comment by aler9** [comment]
This should be fixed in v0.9.3. Please let me know if it worked.

**Comment by QuantumEntangledAndy** [comment]
This now suceeds in connecting the publishing client. I am not able to play (from ffplay or vlc) as it crashes with the following:

```
panic: runtime error: slice bounds out of range [:12846] with capacity 2048

goroutine 8 [running]:
main.(*serverUdpListener).write(...)
	/s/server-udpl.go:96
main.(*program).forwardFrame(0xc00022a070, 0xc000292158, 0x4, 0x0, 0x0, 0xc000300000, 0x322e, 0x80000)
	/s/main.go:525 +0x8e4
main.(*program).run(0xc00022a070)
	/s/main.go:408 +0x1e8
created by main.newProgram
	/s/main.go:264 +0xbb1
```

Probably a different bug I will search for a similar issue and then if I cannot find it open a new issue report
